src/python/grpcio/README.rst

```diff
@@ -50,10 +50,43 @@ Troubleshooting
 Help, I ...
 
 * **... see a** :code:`pkg_resources.VersionConflict` **when I try to install
-  grpc!**
+  grpc:**
```

Comment:
nit: a colon implies a follow up phrase by the same speaker. Here, the voice is that of the user, but what follows is our voice. If the format of the 'trouble' wasn't in the first-person form from the perspective of the user, I might think a colon is fine. As it stands, though, this should be something that maintains consistency with our empathetic use of the user's voice.